src/common/interface.js

```diff
@@ -61,7 +61,7 @@ export function onFocusChanged(callback) {
 –––––––––––––––––––––––––––––––––––––––––––––––––– */
 export function sendMessage(extensionId = null, message, cb) {
```

Comment:
is `extensionId` still needed here?

Reply:
It isn't, but since we have the API shape used around the app it felt safer to leave it in until the next full QA cycle.  I will update this across the app since we are now approaching that.  However, since this already went out as a hot-fix I think we should create a separate PR to make this change across the code base.

Reply:
I can get behind that. This can go live as is and we'll revisit.